wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,303 @@
+| ![][flag_KR] [Reimu-Desu](https://osu.ppy.sh/users/948713) | 1 |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | 1 |
+| ![][flag_JP] [rrtyui](https://osu.ppy.sh/users/352328) | 1 |
+| ![][flag_KR] [sayonara-bye](https://osu.ppy.sh/users/713266) | 1 |
+| ![][flag_FI] [thelewa](https://osu.ppy.sh/users/475021) | 1 |
+| ![][flag_TW] [Uan](https://osu.ppy.sh/users/147623) | 1 |
+| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | 1 |
+
+## Longest Combined Reign Length
+
+*Note: Only reigns after 2011-04-23 are included.*
+
+The following is a list of the total amount of days that a player in the previous lists has had the Rank #1 spot.
+
+| Player | Length (Days) |
+| --: | :-: |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 926 |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 579 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 467 |
+| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | >456 |
+| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 242 |
+| ![][flag_CL] [Mathi](https://osu.ppy.sh/users/5339515) | 212 |
+| ![][flag_JP] [rrtyui](https://osu.ppy.sh/users/352328) | 144 |
+| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 132 |
+| ![][flag_US] [Vaxei](https://osu.ppy.sh/users/4787150) | 77 |
+| ![][flag_KR] [Reimu-Desu](https://osu.ppy.sh/users/948713) | 70 |
+| ![][flag_KR] [FlyingTuna](https://osu.ppy.sh/users/9224078) | 59 |
+| ![][flag_KR] [sayonara-bye](https://osu.ppy.sh/users/713266) | 58 |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | 38 |
+| ![][flag_KR] [firebat92](https://osu.ppy.sh/users/1777162) | 33 |
+| ![][flag_DE] [ShadowSoul](https://osu.ppy.sh/users/494970) | 20 |
+| ![][flag_US] [Freddie Benson](https://osu.ppy.sh/users/7342622) | 19 |
+| ![][flag_TW] [Uan](https://osu.ppy.sh/users/147623) | 11 |
+| ![][flag_FI] [thelewa](https://osu.ppy.sh/users/475021) | 1 |
+| ![][flag_US] [geckogates](https://osu.ppy.sh/users/252524) | <1 |
+
+## Longest Individual Reigns
+
+*Note: Only reigns over 100 days after 2011-04-23 are included.*
+
+The following is a list of how many consecutive days a player from the previous lists has held the Rank #1 spot.
+
+| Player | Reign Length (Days) |
+| --: | :-: |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 460 |
+| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | >456 |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 256 |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 242 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 224 |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 173 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 167 |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 145 |
+| ![][flag_JP] [rrtyui](https://osu.ppy.sh/users/352328) | 144 |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 142 |
```

Comment:
This needs to be fixed again but I'm too dehydrated, hungry, and tired to do it right now. I might do it later if you don't get to it first.